python/python/glide/async_commands/redis_modules/json/commands.py

```diff
@@ -0,0 +1,109 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+import json
+from typing import Dict, List, Optional, Union, cast
+
+from glide.async_commands.core import ConditionalChange
+from glide.constants import TOK
+from glide.protobuf.redis_request_pb2 import RequestType
+from glide.redis_client import TRedisClient
+
+TJson = Union[str, int, float, bool, None, List["TJson"], Dict[str, "TJson"]]
+
+
+class Json:
+    """Class for RedisJson commands."""
+
+    async def set(
+        self,
+        client: TRedisClient,
+        key: str,
+        path: str,
+        value: TJson,
+        set_condition: Optional[ConditionalChange] = None,
+    ) -> Optional[TOK]:
+        """
+        Sets the JSON value at the specified `path` stored at `key`.
+        See https://redis.io/commands/json.set/ for more details.
+
+        Args:
+            client (TRedisClient): The Redis client to execute the command.
+            key (str): The key of the JSON document.
+            path (str): Represents the path within the JSON document where the value will be set.
+                The key will be modified only if `value` is added as the last child in the specified `path`, or if the specified `path` acts as the parent of a new child being added.
+            value (TJSON): The value to set at the specific path.
+            set_condition (Optional[ConditionalChange]): Set the value only if the given condition is met (within the key or path).
+                Equivalent to [`XX` | `NX`] in the Redis API. Defaults to None.
+
+        Returns:
+            Optional[TOK]: If the value is successfully set, return OK.
+                If value isn't set because of `set_condition`, return None.
+
+        Examples:
+            >>> await json_set(client, "doc", "$", {'a': 1.0, 'b': 2})
```

Comment:
Can you expand json examples to be more informative, for example:
```
import json
value = {'a': 1.0, 'b': 2}
str_value = json.dumps(value)
await json.set(client, "doc", "$", value_str)
'OK'  # Indicates successful setting of the value at path '$' in the key stored at 'doc'.
```
and for json.get show how you use json to decode the response to python object

Reply:
(we can have a single example and remove the second you did here)